types/react-jsonschema-form/index.d.ts

```diff
@@ -244,6 +245,10 @@ declare module "react-jsonschema-form" {
     type FormSubmit<T> = {
         formData: T;
     };
+
+    export function withTheme<T>(
+        themeProps: Omit<FormProps<T>, 'schema'>
```

Comment:
Note for myself:  I haven't used `withTheme` myself, but, based on the example [here]( https://react-jsonschema-form.readthedocs.io/en/latest/theme-customization/#withtheme-higher-order-component):
- in the example, `schema` and `uiSchema` are provided to the "wrapped" component
- suiSchema is optional

So I think this should work 👍

Reply:
seems like `Omit` is not defined, at least my typescript complains:
```
Failed to compile
/project-path/node_modules/@types/react-jsonschema-form/index.d.ts
TypeScript error in /project-path/node_modules/@types/react-jsonschema-form/index.d.ts(239,39):
Cannot find name 'Omit'.  TS2304

    237 |     };
    238 | 
  > 239 |     export type ThemeProps<T = any> = Omit<FormProps<T>, 'schema'>;
        |                                       ^
    240 | 
    241 |     export function withTheme<T = any>(
    242 |         themeProps: ThemeProps<T>
```

Reply:
Omit is included on Typescript 3.5

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html#the-omit-helper-type

Reply:
Weird, I'm running typescript 3.5...
I only found this note (below [this example)](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html#example-4) while researching

Reply:
Yes, because on Typescript 2.8, Omit isn't included. But you don't have any problem executing with Typescript 3.5.

Reply:
Make sure that you are running typescript 3.5